docs/en/instructions/quickstart-en.md

```diff
@@ -2,9 +2,8 @@
 
 ## 1、Project orientation
 
-- Eventmesh is a dynamic cloud native event driven infrastructure. It is used to separate the application and back-end middleware layer and it is a cloud native infrastructure as an intermediate layer between the business layer and the infrastructure layer. It can realize automatic capacity expansion and centralized event processing. It can also realize distributed systems under different cloud primitives and event storage of messages, In the future, more features will be supported, such as cloud service features, data real-time computing, etc.
+- Eventmesh is an event driven basic service. As a dynamic plug-in cloud native basic service layer, eventmesh separates the application layer from the middleware layer, and provides flexible, reliable and fast event distribution capabilities. At the same time, it can manage events and serve as the connection layer of application processes, Provide a complete set of application interprocess communication modes required by enterprises to achieve their digital transformation goals.
```

Comment:
Since in the front you use `EventMesh`, so it's better to use `EventMesh` here rather than `Eventmesh` or `eventmesh`.

Reply:
OK.